.vscode/extensions.json

```diff
@@ -1,7 +1,8 @@
 {
-    "recommendations": [
-        "jackmacwindows.craftos-pc",
-        "jackmacwindows.vscode-computercraft",
-        "sumneko.lua"
-    ]
+	"recommendations": [
```

Comment:
We should enable prettier for json formatting in our `.vscode/settings.json` and turn on format on save if it's not already